packages/neuron-ui/src/components/Router/index.tsx

```diff
@@ -123,7 +123,7 @@ export const mainContents: CustomRoute[] = [
   },
   {
     name: `WalletEditor`,
-    path: `${Routes.WalletEditor}/:name`,
+    path: `${Routes.WalletEditor}/:wallet`,
```

Comment:
Are we using wallet id (`identifier`) when constructing this URL? If not (for example if we're using wallet name) please update to make sure we pass wallet identifier instead of name. This also applies to other routes and views where we need to pass or set wallet.

Reply:
Using the whole wallet object now.There is id in object